packages/arb-bridge-peripherals/contracts/tokenbridge/libraries/aeERC20.sol

```diff
@@ -19,12 +19,12 @@
 pragma solidity ^0.6.11;
 
 import "@openzeppelin/contracts-upgradeable/drafts/ERC20PermitUpgradeable.sol";
-import "./ERC677Token.sol";
+import "./TransferAndCallToken.sol";
 
 /// @title Arbitrum extended ERC20
 /// @notice The recommended ERC20 implementation for Layer 2 tokens
-/// @dev This implements the ERC20 standard with extensions to improve UX (ERC677 & ERC2612)
-contract aeERC20 is ERC20PermitUpgradeable, ERC677Token {
+/// @dev This implements the ERC20 standard with callback hook extenstion
```

Comment:
I don't like the idea of calling it a callback hook.
I would describe it as "This implements the ERC20 standard with transferAndCall extenstion/affordances